Cargo.toml

```diff
@@ -36,26 +36,34 @@ tag-name = "v{{version}}"
 # Config for 'cargo dist'
 [workspace.metadata.dist]
 # The preferred cargo-dist version to use in CI (Cargo.toml SemVer syntax)
-cargo-dist-version = "0.6.2"
+cargo-dist-version = "0.8.0"
 # The preferred Rust toolchain to use in CI (rustup toolchain syntax)
 rust-toolchain-version = "1.74.0"
 # CI backends to support
 ci = ["github"]
 # Target platforms to build apps for (Rust target-triple syntax)
 targets = [
-    "x86_64-unknown-linux-gnu",
     "aarch64-apple-darwin",
+    "aarch64-unknown-linux-gnu",
     "x86_64-apple-darwin",
+    "x86_64-unknown-linux-gnu",
     "x86_64-pc-windows-msvc",
 ]
 # Build only the required packages, and individually
 precise-builds = true
 # The archive format to use for non-windows builds (defaults .tar.xz)
 unix-archive = ".tar.xz"
 # The installers to generate for each app
-installers = ["shell", "powershell"]
+installers = ["shell", "powershell", "homebrew"]
 # Publish jobs to run in CI
 pr-run-mode = "plan"
+# A GitHub repo to push Homebrew formulas to
+tap = "probe-rs/homebrew-probe-rs"
+# Publish jobs to run in CI
+publish-jobs = ["homebrew"]
+# Select custom runner for ARM Linux
+[workspace.metadata.dist.github-custom-runners]
+aarch64-unknown-linux-gnu = "buildjet-8vcpu-ubuntu-2204-arm"
```

Comment:
Do we really need an expensive runner for this? Maybe the 2vcpu one would be enough?

And is it possible to use Ubuntu 20.04? We use that for the other builds, for better compatibility.